src/mito2/src/region/opener.rs

```diff
@@ -218,7 +226,8 @@ impl RegionOpener {
         config: &MitoConfig,
         wal: &Wal<S>,
     ) -> Result<Option<MitoRegion>> {
-        let region_options = RegionOptions::try_from(&self.options)?;
+        // Safety: must exist.
+        let region_options = self.options.as_ref().unwrap().clone();
```

Comment:
We don't ensure the options must exist. The opener allows users not to set options before this PR. It uses default options.

Should we initialize options by `RegionOptions::default()`? Otherwise, we should document that the opener will panic if options are unset in a `Panics` section.

Reply:
Added the document.